#! /bin/sh /usr/share/dpatch/dpatch-run
## 32_papersize_b1.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: xfig -papersize b1 now really uses B1 instead of B10
## DP: Closes: #535181

@DPATCH@
diff -urNad xfig~/resources.c xfig/resources.c
--- xfig~/resources.c
+++ xfig/resources.c
@@ -228,7 +228,6 @@
     {"A2      ", "ISO A2  ( 420mm x  594mm)", 19843, 28063}, 
     {"A1      ", "ISO A1  ( 594mm x  841mm)", 28063, 39732}, 
     {"A0      ", "ISO A0  ( 841mm x 1189mm)", 39732, 56173}, 
-    {"B10     ", "JIS B10 (  32mm x   45mm)",  1516,  2117},
     {"B9      ", "JIS B9  (  45mm x   64mm)",  2117,  3017},
     {"B8      ", "JIS B8  (  64mm x   91mm)",  3017,  4300},
     {"B7      ", "JIS B7  (  91mm x  128mm)",  4300,  6050},
@@ -239,6 +238,7 @@
     {"B2      ", "JIS B2  ( 515mm x  728mm)", 24333, 34400},
     {"B1      ", "JIS B1  ( 728mm x 1030mm)", 34400, 48666},
     {"B0      ", "JIS B0  (1030mm x 1456mm)", 48666, 68783},
+    {"B10     ", "JIS B10 (  32mm x   45mm)",  1516,  2117},
     };
 
 char    *multiple_pages[] = {
